definitions/npm/marked-man_v0.1.x/flow_v0.28.x-/marked-man_v0.1.x.js

```diff
@@ -0,0 +1,64 @@
+import marked from 'marked'
```

Comment:
Unfortunately this doesn't work. We haven't yet implemented support for dependencies between libdefs (see also https://github.com/flowtype/flow-typed/issues/16). (As-is, this will just turn out as effectively an `any` type)

For now your best bet is going to be to copy/paste from `marked` into this libdef until we add property support for `declare import` inside `declare module` bodies.